src/Components/ArticleId.jsx

```diff
@@ -60,26 +68,40 @@ function ArticleId() {
   }
 
   function handlePostComment(event) {
-    event.preventDefault()
-    setIsPosting(true)
-    
-    
+    event.preventDefault();
+    setIsPosting(true);
+
     postComment(article_id, user.username, newComment)
       .then((postedComment) => {
         setComments((prevComments) => {
-         return [postedComment, ...prevComments]
-        })
-        setNewComment("")
-        setIsPosting(false)
+          return [postedComment, ...prevComments];
+        });
+        setNewComment("");
+        setIsPosting(false);
       })
-      
+
       .catch((error) => {
-        setPostError(true)
-        setIsPosting(false)
+        setPostError(true);
+        setIsPosting(false);
+      });
+  }
+
+  function handleDeleteComment(comment_id) {
+    setDeletingCommentId(comment_id);
+
+    deleteComment(comment_id)
+      .then(() => {
+        setComments((prevComments) =>
+          prevComments.filter((comment) => comment.comment_id !== comment_id)
+        );
+        setDeletingCommentId(null);
```

Comment:
So you have declared the deletingCommentId state but never use it in this file. You could remove it from the function and it would still work as normally as you use the comment_id passed to the function to perform the delete instead of the the deletingCommentId , I would just remove the state.